kalite/store/models.py

```diff
@@ -1,16 +1,8 @@
 import datetime
```

Comment:
@jamalex Reckon we can just jettison the store app completely? I don't see any need to keep the models around as we will no longer need to sync the data.